develop/api_udf.rst

```diff
@@ -101,8 +101,9 @@ Undistributing moves all data from shards back into a local table on the
 coordinator node (assuming the data can fit), then deletes the shards.
 
 Citus will not undistribute tables that have -- or are referenced by -- foreign
-keys. Before undistributing a table you must drop its foreign key constraints,
-or those in other tables pointing to it.
+keys unless `cascade_via_foreign_keys` option is set to `true`.
+If `cascade_via_foreign_keys` is set to `false` (by default), you must drop its
+foreign key constraints, or those in other tables pointing to it before undistributing.
```

Comment:
I think it'd be good to mention what happens when `cascade_via_foreign_keys ` = true.  That is a case where users might be careful. `local < ref < dist` case, we'd even undistribute the `dist`, right?

Reply:
true, I will explicitly mention that too